cmd/wallet-web/src/pages/WACIIssue.vue

```diff
@@ -329,11 +258,11 @@ export default {
       }
 
       console.debug('acknowledgment msg', ack);
-      let { status, url } = ack;
-
-      this.loading = false;
-      // TODO [Issue#1336]  check if status="FAIL", then should redirect to generic error screen, it means WACI flow didn't succeed
-      this.protocolHandler.done(url);
+      const { status, url } = ack;
+      this.redirectUrl = url;
+      // TODO: issue#1336 - check if status="FAIL", then should redirect to generic error screen, it means WACI flow didn't succeed
```

Comment:
I see you changed TODO issue number pattern to `issue1336` at one place and `issue#136` another place. We usually follow pattern of `[Issue#<NUMBER>]` format so that it can be easy to locate/search.

Reply:
Sure, let's document this format so that everybody is on the same page 👍